### DISCUSSION TITLE
Search for passwords

### DISCUSSION THREAD

**Discussion by HassanMullah** [dani-garcia/vaultwarden#1291] [Ideas]
Hello, I have always used KeePass 2 for the last few years, but I find the integration of Bitwarden in the browsers better solved, which is why I am now starting to switch.

But what I miss is the search for passwords. It would make sense to see where people still use the same passwords and then break them down into individual passwords.

**Comment by brent3000** [comment]
Coming from KP2 the search function is on the vault page, just start to type and it will filter the results? Not sure if thats what you are referring to,

Regarding the same password under report is the tools which show that data for the same password being used etc

**Reply by HassanMullah** [reply]
I discovered the search function long ago, but Bitwarden does not search for passwords here. I don't want a report that tells me that this or that password has already been used x times, but I would like to see all entries where this password I am looking for is used.

**Comment by jjlin** [comment]
Bitwarden doesn't currently support password searching, but it's also something that would have to be handled within the clients, not a backend implementation like bitwarden_rs.

You can vote for this feature at https://community.bitwarden.com/t/search-by-password/11941. In the meantime, one possible workaround is to export your vault to CSV or JSON and search for the password there.
